features/attached_commands.feature

```diff
@@ -70,7 +70,7 @@ Feature: Command behaviour when attached to an UA subscription
            | xenial  |
 
     @series.all
-    Scenario Outline: Attached detach in a trusty machine
+    Scenario Outline: Attached detach in a ubuntu machine
```

Comment:
Since you are referencing staging_token here you need the appropriate decorator too.
```suggestion
   @uses.config.contract_token_staging
    Scenario Outline: Attached detach in a ubuntu machine
```